.travis.yml

```diff
@@ -65,19 +65,6 @@ env:
 
 matrix:
   include:
-    - python: 2.7
```

Comment:
I've got rid of the py27 build to avoid having to do funny things with importing `signature`.